Sources/SwiftSyntaxBuilderGeneration/String+Extensions.swift

```diff
@@ -14,4 +14,5 @@ import Foundation
 
 extension StringProtocol {
     var withFirstCharacterLowercased: String { prefix(1).lowercased() + dropFirst() }
+    var backticked: String { "`\(replacingOccurrences(of: "`", with: ""))`" }
```

Comment:
Why do we need `replacingOccurrences(of: "`", with: "")`? Is `token.name` sometimes already backticked?

Reply:
I don't think so, my thinking was to make `.backticked` idempotent. But yeah, it would probably be simpler to just remove it.

Reply:
I think just surrounding the text with `` ` `` is the most intuitive behavior, in particular if we don’t need idempotency.